newIDE/app/src/MainFrame/Preferences/PreferencesDialog.js

```diff
@@ -138,14 +138,15 @@ const PreferencesDialog = ({ i18n, onClose }: Props) => {
                     );
                     event.preventDefault();
                   }}
+                  color="secondary"
```

Comment:
Yes, the links on the design are meant to be with secondary color, otherwise they're not readable.
Should I create a UI Component that does it for us? (instead of importing MUI one)

Reply:
That's ok but definitely something to refactor at some point, so we can just use `Link` everywhere and don't worry about a future developer being "oops I forgot the `color="secondary"`, but if it's used everywhere why wasn't this factored? Are there links with *another* color?"